tools/twix/src/panels/remote.rs

```diff
@@ -0,0 +1,89 @@
+use std::sync::Arc;
+
+use eframe::egui::Widget;
+use gilrs::{Axis, Button, Gamepad, GamepadId, Gilrs};
+use serde_json::{json, Value};
+use types::Step;
+
+use crate::{nao::Nao, panel::Panel};
+
+pub struct RemotePanel {
+    nao: Arc<Nao>,
+    gilrs: Gilrs,
+    active_gamepad: Option<GamepadId>,
+    enabled: bool,
+}
+
+impl Panel for RemotePanel {
+    const NAME: &'static str = "Remote";
+
+    fn new(nao: Arc<Nao>, _value: Option<&Value>) -> Self {
+        let gilrs = Gilrs::new().unwrap();
```

Comment:
maybe you can present a better error message if the `Gilrs` creation failed?

Reply:
Changed to expect with a message explaining what happened.